README.md

```diff
@@ -107,4 +107,6 @@ $ nodemon bin/www
 
 ## Contributing
 
-See the [Contributing](contributing.md) guide to contribute to the project!
\ No newline at end of file
+See the [Contributing](contributing.md) guide to contribute to the project!
+
+[![Build Node](https://github.com/ChicoState/PantryNode/actions/workflows/actions.yml/badge.svg)](https://github.com/ChicoState/PantryNode/actions/workflows/actions.yml)
```

Comment:
This should replace lines 22~26

Reply:
Replaced Lines 22-26 with Github actions status badge